chsdi/tests/functional/test_helpers.py

```diff
@@ -77,3 +88,46 @@ def test_transformCoordinate(self):
         self.assertTrue(isinstance(wkt_21781, Geometry))
         self.assertEqual(int(wkt_21781.GetX()), 595324)
         self.assertEqual(int(wkt_21781.GetY()), 84952)
+
+    def test_check_even(self):
+        testnumber = 10
+        result = check_even(testnumber)
+        self.assertEqual(result, True)
+
+        testnumber = 5
+        result = check_even(testnumber)
+        self.assertEqual(result, False)
+
+    def test_round(self):
+        testnumber = 4.4
+        result = round(testnumber)
+        self.assertTrue(result == 4, result)
+
+    def test_format_search_text(self):
+        testinput_str = '//'
+        result = format_search_text(testinput_str)
+        self.assertTrue(result, False)
```

Comment:
It's assertTrue here, not assertEqual. So the False has no impact and test can never faile.